utils.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
error executing this script.
what is its utility ?

Reply:
The documentation for this script will be updated in the README and the ecoCode-common project today.

Reply:
ok seen together